components/function-controller/internal/controllers/serverless/function_reconcile_test.go

```diff
@@ -1332,7 +1332,7 @@ func TestFunctionReconciler_Reconcile(t *testing.T) {
 
 		function = &serverlessv1alpha2.Function{}
 		g.Expect(function.Spec.RuntimeImageOverride).To(gomega.Equal(""))
-		g.Expect(function.Status.RuntimeImageOverride).To(gomega.Equal(""))
+		g.Expect(function.Status.RuntimeImage).To(gomega.Equal(""))
```

Comment:
Shouldn't be there `fn-image` from configmap?

Reply:
There is no `fn-image` in the configmap so I think this is correct